### ISSUE TITLE
GLIBC 2.7 not found

### ISSUE THREAD

**Issue by opnoack** [haiwen/seafile#1364]
Seafile-Cli: seafile-cli_4.3.2_i386.tar.gz

System: Ubuntu 7.10

Expected behaviour: seaf-cli start should succeed

Actual behaviour: seaf-cli doesn't start and gives the following errors

```
starting ccnet daemon ...
ccnet: /lib/tls/i686/cmov/libc.so.6: version `GLIBC_2.7' not found (required by ccnet)
ccnet: /lib/tls/i686/cmov/libc.so.6: version `GLIBC_2.7' not found (required by /opt/seafile-cli-4.3.2/lib/libevent-2.0.so.5)
ccnet: /lib/tls/i686/cmov/libc.so.6: version `GLIBC_2.7' not found (required by /opt/seafile-cli-4.3.2/lib/libcrypto.so.10)
ccnet: /lib/tls/i686/cmov/libc.so.6: version `GLIBC_2.9' not found (required by /opt/seafile-cli-4.3.2/lib/libgio-2.0.so.0)
ccnet: /lib/tls/i686/cmov/libc.so.6: version `GLIBC_2.7' not found (required by /opt/seafile-cli-4.3.2/lib/libglib-2.0.so.0)
ccnet: /lib/tls/i686/cmov/libc.so.6: version `GLIBC_2.9' not found (required by /opt/seafile-cli-4.3.2/lib/libglib-2.0.so.0)
ccnet: /lib/tls/i686/cmov/libc.so.6: version `GLIBC_2.7' not found (required by /opt/seafile-cli-4.3.2/lib/libgssapi_krb5.so.2)
ccnet: /lib/tls/i686/cmov/libc.so.6: version `GLIBC_2.7' not found (required by /opt/seafile-cli-4.3.2/lib/libkrb5.so.3)
ccnet: /lib/tls/i686/cmov/libc.so.6: version `GLIBC_2.7' not found (required by /opt/seafile-cli-4.3.2/lib/libkrb5support.so.0)
CCNet daemon failed to start.
```

**Comment by shoeper** [comment]
Your ubuntu version is too old (8 years). So don't think someone is going to create a fix.

Maybe this helps: http://ubuntuforums.org/showthread.php?t=1369339

**Comment by opnoack** [comment]
I know this version is really old. But at least you could have pointed it out in the changelog. If there will be no fix for this I will also be stuck with #1365.

**Comment by freeplant** [comment]
We will not support old Ubuntu system.
